### DISCUSSION TITLE
Strapi - Prorperty of Causes listing

### DISCUSSION THREAD

**Discussion by smsalmanshahAN** [an-nadaaa/nda-front#125] [Feedback]
The lisitng of causes should have the title of the cause as the first one to easily identify the cause

![image](https://user-images.githubusercontent.com/101787508/161202664-82caca28-3a61-4ccc-b9b0-cbac37324dc2.png)

**Comment by an-nadaa** [comment]
@smsalmanshahAN This was fixed.
